crates/bvs-registry/src/state.rs

```diff
@@ -169,6 +197,161 @@ pub fn decrease_operator_active_registration_count(
     })
 }
 
+#[cw_serde]
+pub struct SlashingRegistry {
```

Comment:
```suggestion
#[cw_serde]
pub struct SlashingParameters {
```

I think calling it slashing parameters is more purposeful.